### PR TITLE
Added a license to the entire repo.

### PR DESCRIPTION
**Pull request by toji**
**TL;DR: If you have contributed to the webvr.info repository please respond in this pull request that you consent to have the standard Chromium license applied.**

It's been pointed out that (due to some quirks of how this repo came to be and my own inattentiveness) the top level of the repo does not have a proper license file. (The samples directory does, primarily because it started life as a separate repo.) I'd like that correct that so that we can make easier use of the full repo in things like test environments. As such, this pull request will apply the same standard Chromium license that was already applied to the samples directory to the entire repository.

Given the somewhat touchy nature of licensing things, however, I'm reaching out to anyone who has previously contributed to the non-samples portion of the repo to get written consent that you're OK with having the license applied. If not we will have to remove and replace your contributions. Sorry!

Thanks for everyone's help, and sorry for the inconvenience.

Need written consent from:
@mkeblx, @HalfdanJ, @cvan, @peterood, @willeastcott, @poshaughnessy, @ngokevin, @leweaver, @kschzt, @brianpeiris, @anssiko 

And for the record, I consent to this change as well. :)

### CODE REVIEW

**Comment by leweaver** [comment]
Thanks for adding a license :)

I will need to check internally - The Chromium License isn't one that I can auto-consent to. I'll get back to you...

**Comment by HalfdanJ** [comment]
Good call. I consent

**Comment by ngokevin** [comment]
Kk I consent

**Comment by peterood** [comment]
I consent 👍🏼

**Comment by willeastcott** [comment]
I consent to this change.

**Comment by mkeblx** [comment]
I consent.

**Comment by brianpeiris** [comment]
👌 I consent

**Comment by toji** [comment]
Half way there! Thanks to everyone who's responded already.

Still waiting for: @cvan, @poshaughnessy, @kschzt, and @anssiko.
(and for @leweaver to talk to the Microsoft lawyers. 😉)

**Comment by poshaughnessy** [comment]
I consent! 👍

**Comment by kschzt** [comment]
I consent!

Best,

Jaakko

**Comment by toji** [comment]
Ping @cvan, @anssiko, and @leweaver

**Comment by leweaver** [comment]
Still waiting on internal confirmation here ;)

**Comment by leweaver** [comment]
Consent!

**Comment by toji** [comment]
FYI: After some further digging it appears that @anssiko only contributed a single link URL update, which has since been replaced by a new link in subsequent code changes, so his consent isn't required.

That leaves @cvan as the final outstanding confirmation.

**Comment by cvan** [comment]
I consent to this license addition. Thanks, @toji.

**Comment by toji** [comment]
Awesome, thanks everyone!

**Comment by anssiko** [comment]
Consent.

(Sorry for the delayed response, I was OoO.)
